modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -777,13 +783,19 @@ void ONNXCompiled::Run(const std::vector<cv::Mat>& ins,
 
         GAPI_Assert(outputs.size() == out_names.size());
         // Fill in ONNX tensors
-        for (auto &&iter : ade::util::zip(ade::util::toRange(out_tensor_info),
-                                          ade::util::toRange(outputs))) {
-            const auto &out_name   = std::get<0>(iter).name;
-                  auto &out_tensor = std::get<1>(iter);
-            onnx_outputs[out_name] = toCV(out_tensor);
+        if (params.names_to_remap.empty()) {
```

Comment:
Is it possible to merge this condition with the above one

Reply:
Merged.